### ISSUE TITLE
Napari viewer runs with notebook but not within the exported script with TypeError: ChannelABC() takes no arguments

### ISSUE THREAD

**Issue by sebi06** [napari/napari#1054]
## 🐛 Bug

Hi guys,

i just convert a jupyter notebook into a *.py file and tried to run it in VS Code on Ubuntu and got errors. But from with the original notebook everything works fine. Any ideas?

```bash
[Running] python -u "/datadisk1/tuxedo/Github/napari_x_pylibczi/example.py"
[{'X': (0, 1086), 'Y': (0, 832), 'Z': (0, 23), 'C': (0, 2), 'T': (0, 30), 'B': (0, 1)}]
BTCZYX
Traceback (most recent call last):
  File "/datadisk1/tuxedo/Github/napari_x_pylibczi/example.py", line 156, in <module>
    napari.view_image(img)
  File "/home/sebi06/miniconda3/lib/python3.7/site-packages/napari/view_layers.py", line 116, in view_image
    title=title, ndisplay=ndisplay, order=order, axis_labels=axis_labels
  File "/home/sebi06/miniconda3/lib/python3.7/site-packages/napari/viewer.py", line 60, in __init__
    qt_viewer = QtViewer(self)
  File "/home/sebi06/miniconda3/lib/python3.7/site-packages/napari/_qt/qt_viewer.py", line 54, in __init__
    self.console = QtConsole({'viewer': self.viewer})
  File "/home/sebi06/miniconda3/lib/python3.7/site-packages/napari/_qt/qt_console.py", line 43, in __init__
    kernel_client.start_channels()
  File "/home/sebi06/miniconda3/lib/python3.7/site-packages/qtconsole/kernel_mixins.py", line 47, in start_channels
    super(QtKernelClientMixin, self).start_channels(*args, **kw)
  File "/home/sebi06/miniconda3/lib/python3.7/site-packages/ipykernel/inprocess/client.py", line 64, in start_channels
    super(InProcessKernelClient, self).start_channels()
  File "/home/sebi06/miniconda3/lib/python3.7/site-packages/jupyter_client/client.py", line 118, in start_channels
    self.control_channel.start()
  File "/home/sebi06/miniconda3/lib/python3.7/site-packages/jupyter_client/client.py", line 200, in control_channel
    socket, self.session, self.ioloop
TypeError: ChannelABC() takes no arguments

[Done] exited with code=1 in 1.754 seconds
```

**Comment by sofroniewn** [comment]
This is likely caused by a recent `jupyter_client` upgrade - we've pinned to prevent it, see this quick fix here https://github.com/napari/napari/pull/997. But now maybe a good time to look into a better solution.

**Comment by sofroniewn** [comment]
Note this has been reported at https://github.com/jupyter/jupyter_client/issues/523 and there is a PR fix at https://github.com/ipython/ipykernel/pull/489 that is waiting for merge and then release

**Comment by sofroniewn** [comment]
For now best to just try using napari with `jupyter-client-5.3.4` instead of `jupyter-client-6.0.0`

```
pip uninstall jupyter-client
pip install jupyter-client==5.3.4
```

**Comment by sofroniewn** [comment]
This was closed by #1240
